### PR TITLE
types: improve activity data point, add support for ac power

### PR DESCRIPTION
**Pull request by abn**

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -387,14 +387,30 @@ export type DataPointTemperature = Temperature & {
...
   precision: Temperature;
+} & {
+  value?: Power;
 };
 
-export type DataPoint = DataPointPercentage | DataPointTemperature;
-
-export type ZoneActivityDataPoints = {
-  heatingPower: DataPoint;
+export type DataPointPower = {
```
Comment (mattdavis90) [inline src/types/index.ts:394]: How do you feel about this being a `PowerDataPoint` to match the heating one?

Comment (abn) [reply src/types/index.ts:394]: Will fix it

**Comment by mattdavis90** [comment]
Oh. I just noticed that all the others are the other way around. `DataPointTemperature` etc. I should have probably checked that properly before jumping to the other way.

How does your OCD feel about the need to align them? Mine's twitching.

**Comment by abn** [comment]
Happy to align them in the PR.

**Comment by abn** [comment]
Done

**Comment by mattdavis90** [comment]
Thanks - sorry for being fussy. I really appreciate the effort you're putting into the library!

**Comment by abn** [comment]
I really just want a canonical source for this API :D so, happy to help. Right now it aligns with the work I am doing at https://github.com/abn/com.tado.community.home/ - so it is a win win.

**Pull request merged**